videostore/videostore.go

```diff
@@ -295,13 +295,19 @@ func (vs *videostore) Segmenter() *RawSegmenter {
 }
 
 func (vs *videostore) Fetch(_ context.Context, r *FetchRequest) (*FetchResponse, error) {
+	// Convert incoming local times to UTC for consistent timestamp handling
+	// All internal operations and stored timestamps are in UTC
+	r.From = r.From.UTC()
```

Comment:
Given the code to take the docommand time range strings and turn them into time.Time lives in the caller of Fetch and Save I recommend making this change in the caller rather than here.

Reply:
Moved the conversion to a new helper `parseTimeRange`

Reply:
Undid the .UTC conversion (made it go back in Save and Fetch), but kept the helper for DRY.